### PR TITLE
Hide realtime status if we don't have a stop name

### PR DESCRIPTION
**Pull request by arkadyan**
#### Summary of changes
**Asana Ticket:** [Polish | Map tooltip looks broken when trip assignment is missing](https://app.asana.com/0/555089885850811/1180626064629737)

Hide realtime status if we don't have a stop name in vehicle tooltips on the line diagram and map.

Screen Shot 2020-07-09 at 11 35 47

Screen Shot 2020-07-09 at 11 57 41

---

Before getting review, please check the following:

* [x] Does frontend functionality render and work correctly in IE?
* [x] Have we load-tested any new pages or internal API endpoints that will receive significant traffic?
* [x] Are interactive elements accessible to screen readers?
* [x] Have you checked for tech debt you can address in the area you're working in?
* [x] If this change involves routes, does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?
* [x] Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?

### CODE REVIEW

File: apps/stops/lib/stop.ex [hunk 1]
```diff
@@ -35,7 +35,7 @@ defmodule Stops.Stop do
           id: id_t,
           parent_id: id_t | nil,
           child_ids: [id_t],
-          name: String.t(),
+          name: String.t() | nil,
```
Comment (thecristen) [inline apps/stops/lib/stop.ex:38]: Not sure if this change is needed, given we appear to be working with an empty string in this case.

File: apps/site/assets/ts/schedule/components/line-diagram/VehicleIcons.tsx [hunk 2]
```diff
@@ -30,10 +30,13 @@ const statusDescriptions = {
 
 const tooltipText = (
   routeType: RouteType | null,
-  stopName: string,
+  stopName: string | null,
   vehicle: LineDiagramVehicle
 ): string => {
-  const status = `${statusDescriptions[vehicle.status]} ${stopName}`;
+  const status =
+    stopName !== null && stopName.length
```
Comment (thecristen) [inline apps/site/assets/ts/schedule/components/line-diagram/VehicleIcons.tsx:37]: Would `stopName && stopName.length` work here?

Comment (arkadyan) [reply apps/site/assets/ts/schedule/components/line-diagram/VehicleIcons.tsx]: Yep! Updated

File: apps/stops/lib/stop.ex (hunk 1 above)
Comment (arkadyan) [reply apps/stops/lib/stop.ex:38]: I can't tell if we are ever getting `nil` in this instance, but it seemed possible and even if not if you look up at the defstruct `name` defaults to `nil`, so this seems more correct.

**Comment by ingridpierre** [comment]
Functionally works as expected, I think our min width on this tooltip causes a strange appearance though. Is there a reason we can't just have a max width for long tooltips and let the smaller ones (in cases like this, esp. without the crowding pill) be small?

image

**Comment by arkadyan** [comment]
@ingridpierre I removed a 320px min width we had set. Does this look better?

Screen Shot 2020-07-10 at 15 47 33

Screen Shot 2020-07-10 at 15 48 26

**Pull request merged**